### ISSUE TITLE
docker no http beyond root

### ISSUE THREAD

**Issue by valentinedwv** [mosuka/blast#136]
I get 404 for any query except /
```
curl -X GET http://localhost:10002/
{
  "status": 200,
  "version": "v0.8.1"
}
```

failure

```
$ curl -X GET http://localhost:10002/v1/liveness_check 
404 page not found
```

```
  blast:
     image: mosuka/blast:v0.8.1
     ports:
       - 10000:10000
       - 10001:10001
       - 10002:10002
     ulimits:
       nofile:
         soft: "65536"
         hard: "65536"
#     env_file:
#       - ./example.env
     environment:
       - SERVICE_PORTS=10000, 10001, 10002
     volumes:
       - "/data/volumes/blast:/data"
#     networks:
#       - web
# 0.3.0
#     command: ["start", "--bind-addr=:10000", "--grpc-addr=:10001", "--http-addr=:10002", "--node-id=node1"]
# v0.8.1
     command: ["blast", "indexer","start","--data-dir=/data", "--node-address=:10000", "--grpc-address=:10001", "--http-address=:10002", "--node-id=node1"]

```

**Comment by pffreitas** [comment]
I had the same issue and upgraded to version 0.9.0 which works fine. It's not published under `mosuka/blast` on docker hub though. I end up building a docker image for 0.9.0 and it's published on ` pffreitas/blast`

**Comment by mosuka** [comment]
@valentinedwv @pffreitas 
Sorry, it took me so long to reply. And I'm sorry for the inconvenience.
I released v0.9.1 and also pushed containers to Docker Hub. Please use it.

Blast uses grpc-gateway. Check the following files for available HTTP endpoints:
https://github.com/mosuka/blast/blob/master/protobuf/index.proto
